app/views/editions/workflow/review.html.erb

```diff
@@ -85,11 +85,31 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      <%= render Shared::ContinueOrCancelButtonGroup.new(
-        form_id: "review",
-        button_text: button_text_submit_for(@edition),
-        edition: @edition,
-        ) %>
+      <div class="govuk-button-group govuk-!-margin-bottom-6">
+        <div>
+          <%= render "govuk_publishing_components/components/button", {
+            text: button_text_submit_for(@edition),
+            name: "confirm",
+            type: "submit",
+            form: "review",
+            data_attributes: {
+              "testid": "submit-button",
+            },
+          } %>
+        </div>
+        <% if current_user.has_permission?(User::Permissions::PRE_RELEASE_FEATURES_PERMISSION) %>
```

Comment:
Not for now, but I wonder if we could break this out into a helper? It's likely to get repetitive!